dips/tests/test_views.py

```diff
@@ -0,0 +1,42 @@
+from django.test import TestCase
+from django.test.client import RequestFactory
+from unittest.mock import ANY, patch
+
+from dips.models import User
+from dips.views import search
+
+
+class ViewsTests(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_superuser(
+            'admin', 'admin@example.com', 'admin')
+        self.factory = RequestFactory()
+
+    @patch('elasticsearch_dsl.Search.execute')
+    @patch('elasticsearch_dsl.Search.count', return_value=0)
+    @patch('dips.views.messages.error')
+    @patch('dips.views.render')
+    def test_search_wrong_dates(self, mock, mock_2, patch, patch_2):
```

Comment:
If you have multiple patches, I find it easier to reason about if you name them meaningfully, e.g. mock_execute, mock_count, mock_error_message, mock_render or similar.

Reply:
Totally agree! I'll create a different PR to address that as it's that way all over the tests.